Trappist/src/Promact.Trappist.Test/Questions/QuestionsRepositoryTest.cs

```diff
@@ -12,12 +14,16 @@ namespace Promact.Trappist.Test.Questions
     [Collection("Register Dependency")]
     public class QuestionsRepositoryTest : BaseTest
     {
-        private readonly IQuestionRespository _questionRepository;
+        private readonly IQuestionRepository _questionRepository;
+        private readonly Mock<IHostingEnvironment> _hostingEnvironmentMock;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the "_hostingEnvironmentMock" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)